core/src/main/java/hera/core/Core.java

```diff
@@ -204,7 +204,10 @@ public static void main(String[] args) {
 								).next()
 						)
 				)
-				.subscribe();
+				.subscribe(aVoid -> Mono.empty(), throwable -> {
+					LOG.debug("Top level error catch block:");
+					LOG.debug("Stacktrace:", throwable);
+				});
```

Comment:
Added this `catch` block here, so we can be sure to log every error that occurs in the main command stream.
I think this also prevents the app from crashing, not 100% sure tho.